conda.recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: dask
+  # GIT_DESCRIBE_TAG is supposed to return the latest version (0.4.0) but it is
+  # not working for some reason. And PKG_VERSION is incorrect (it says 1.0.1k)
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '0.4.0')}}
```

Comment:
Do we know why this is?  Does it work locally and not on binstar?  Perhaps our tags are mismanaged somehow?  If we can't figure this out we should bug the conda folk.